containers/proxy-helm/Chart.yaml

```diff
@@ -7,5 +7,5 @@ description: Uyuni proxy containers.
 type: application
 home: https://www.uyuni-project.org/
 icon: https://www.uyuni-project.org/img/uyuni-logo.svg
-version: 0.0.0 # Only increase the patch number when changing, the rest is replaced when pushing to OBS
+version: 0.0.0
 appVersion: "%PKG_VERSION%"
```

Comment:
Since we want these in sync, why not make both version and appVersion `"%PKG_VERSION"`?

Reply:
if you want to run helm directly from the source code you need a valid `version` there, while `appVersion` is not checked